scripts/deploy/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think we should find a way to have this file get linted by our default linting rules.

Reply:
That's what we're doing actually. These are just the overrides.